ament_clang_tidy/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ament_clang_tidy</name>
+  <version>6.0.0</version>
+  <description>
+    The ability to check code against style conventions using
+    clang-tidy and generate xUnit test result files.</description>
+  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
```

Comment:
Make yourself the maintainer; I think Dirk already has enough :).  Ditto for setup.py

Reply:
+:100: